### ISSUE TITLE
Finding the best result

### ISSUE THREAD

**Issue by reiddouglas** [reiddouglas/skin-disease-cnn#3]
Currently, the cnn_training script iterates over and saves a model for every fold and epoch. However, it will be hard to manually sort through and find the best-performing model. There are two options:
- Create another script for checking each saved model for the best validation loss.
- Within the training script, keep track of the best model.

The first option is more robust since it will work even if there are problems (e.g. a crash) when the training is happening. The second option is quicker and dirtier.
